src/filesystem/directory.rs

```diff
@@ -51,7 +51,7 @@ pub struct DirEntry {
 pub struct RawDirectory(pub(crate) SearchId);
 
 impl RawDirectory {
-    /// Convert a raw file into a droppable [`File`]
+    /// Convert a raw file into a droppable [`crate::filesystem::File`]
```

Comment:
I think it should be fixed like this instead:

```suggestion
    /// Convert a raw directory into a droppable [`Directory`]
```